otomoto/spiders/otomoto.py

```diff
@@ -37,30 +34,44 @@ def parse_car_page(self, response):
             'Marka pojazdu': 'brand',
             'Model pojazdu': 'model',
             'Rok produkcji': 'year',
+            'Wersja': 'version',
             'Przebieg': 'mileage',
             'Pojemność skokowa': 'capacity',
             'Moc': 'horse_power',
             'Rodzaj paliwa': 'fuel_type',
             'Skrzynia biegów': 'transmission',
             'Typ': 'type',
             'Liczba drzwi': 'number_of_doors',
+            'Kraj pochodzenia': 'origin_country',
             'Kolor': 'color',
             'Pierwszy właściciel': 'first_owner',
             'Bezwypadkowy': 'no_accidents',
             'Serwisowany w ASO': 'aso',
             'Stan': 'condition',
         }
         loader = OtomotoCarLoader(OtomotoItem(), response=response)
+
         for params in response.css('.offer-params__item'):
 
             property_name = params.css('.offer-params__label::text').extract_first().strip()
             if property_name in property_list_map:
-                css = params.css('div::text').extract_first().strip()
+                css = params.css('.offer-params__value::text').extract_first().strip()
                 if css == '':
                     css = params.css('a::text').extract_first().strip()
 
                 loader.add_value(property_list_map[property_name], css)
 
         loader.add_css('features', '.offer-features__item::text')
         loader.add_value('url', response.url)
+
+        # add price
+        for price_params in response.css('.price-wrapper'):
+            price_value = price_params.css('.offer-price__number::text').extract_first().replace(" ","")
```

Comment:
Also, can you make it `ItemLoader`?

Reply:
I try to use
`loader.add_css('price', '.offer-price::attr(data-price)'.replace(" ", ""))`
or
`loader.add_css('price', '.offer-price__number::text'.replace(" ",""))`
to remove space but it is not working. I try to use 're' but I dont have idea to ignore spaces.
The loop was the simple way for me. Maybe you got some idea?

Reply:
Sure, I used simple add_css like here
https://github.com/dolohow/scrapy-otomoto/blob/08b9ac29707a29be14e7ff1fca954bcf0ed6bf13/otomoto/spiders/otomoto.py#L71

and later I removed spaces in output filter.